src/snowflake/connector/result_set.py

```diff
@@ -63,41 +63,27 @@ def result_set_iterator(
     """
 
     with ThreadPoolExecutor(prefetch_thread_num) as pool:
-        # Fill up window
-
         logger.debug("beginning to schedule result batch downloads")
-
         for _ in range(min(prefetch_thread_num, len(unfetched_batches))):
             logger.debug(
                 f"queuing download of result batch id: {unfetched_batches[0].id}"
             )
             unconsumed_batches.append(
                 pool.submit(unfetched_batches.popleft().create_iter, **kw)
             )
```

Comment:
This block of code is kept to not break tests, it seems like some tests relies on how chunks are consumed.

Reply:
I'm curious what tests are broken? the logic is the same as line 76-80.